config/helm/chart/default/templates/Common/opentelemetry-collector/clusterole-telemetry-endpoints.yaml

```
@@ -0,0 +1,67 @@
+{{- if .Values.rbac.telemetryService.create }}
+# Copyright 2021 Dynatrace LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: dynatrace-telemetry-endpoints
```

```suggestion
  name: dynatrace-telemetry-service
```
I would align this with the naming in Dynkaube

---

fixed 5895f2134